manga_translator/translators/chatgpt.py

```diff
@@ -136,22 +136,35 @@ async def _request_translation(self, prompt: str) -> str:
         self.token_count_last = response.usage['total_tokens']
         return response.choices[0].text
 
-
-class GPT35TurboTranslator(GPT3Translator):
+class GPTChatTranslator(GPT3Translator):
     _MAX_REQUESTS_PER_MINUTE = 200
     PROMPT_TEMPLATE = SIMPLE_PROMPT_TEMPLATE
+    _RETRY_ATTEMPTS = 5
+    _model: Literal['gpt-3.5-turbo', 'gpt-4']
```

Comment:
I dont see the need of adding linting to something like that as it would only increase the amount of changes needed to be made when adding future models.
That doesnt matter here though as I want you to remove GPTChatTranslator anyways.